src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -238,6 +239,14 @@ class ReimbursementAccountPage extends React.Component {
                             achData={this.props.reimbursementAccount.achData}
                         />
                     )}
+                    <ConfirmModal
+                        title={this.props.translate('companyStep.confirmModalTitle')} // TODO: Update translation key
+                        onConfirm={() => hideErrorModal()}
```

Comment:
No need to use an arrow function for this can just do `onConfirm={hideErrorModal}`

Reply:
done